react-accordion/my-app/src/TopicContent.tsx

```diff
@@ -0,0 +1,7 @@
+type Props = {
+  isDisplayed: boolean;
+  content: string;
+};
+export function TopicContent({ isDisplayed, content }: Props) {
+  return isDisplayed ? <div>{content}</div> : undefined;
```

Comment:
JSX should be `null` if you don't want it to display. `undefined` works, but the spec says it should be `null`.